### ISSUE TITLE
Inline templates rule

### ISSUE THREAD

**Issue by princed** [Gillespie59/eslint-plugin-angular#58]
Rule to prohibit inline templates in directives, when there are escaped quotes in string or multiline strings.

The following patterns are considered warnings:
```js
{
  template: '<button class="btn" ng-click="model = !model">' +
              '<span ng-transclude></span>' +
            '</button>'
}
```

```js
{
  template: '<button ng-class="{ \'btn-big\': isBig  }">Text</button>'
}
```

The following patterns are considered okay and do not cause warnings:

```js
{
  template: '<button>Text</button>'
}
```

```js
{
  template: require('button.html')
}
```

**Comment by princed** [comment]
Awesome! Thank you, @remcohaszing and @Gillespie59!
